sim/shaman/spirit_wolves.go

```diff
@@ -0,0 +1,116 @@
+package shaman
+
+import (
+	"math"
+	"strconv"
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+	"github.com/wowsims/wotlk/sim/core/proto"
+	"github.com/wowsims/wotlk/sim/core/stats"
+)
+
+type SpiritWolf struct {
+	core.Pet
+
+	shamanOwner *Shaman
+}
+
+type SpiritWolves struct {
+	SpiritWolf1 *SpiritWolf
+	SpiritWolf2 *SpiritWolf
+}
+
+func (SpiritWolves *SpiritWolves) EnableWithTimeout(sim *core.Simulation) {
+	SpiritWolves.SpiritWolf1.EnableWithTimeout(sim, SpiritWolves.SpiritWolf1, time.Second*45)
+	SpiritWolves.SpiritWolf2.EnableWithTimeout(sim, SpiritWolves.SpiritWolf1, time.Second*45)
+}
+
+func (SpiritWolves *SpiritWolves) CancelGCDTimer(sim *core.Simulation) {
+	SpiritWolves.SpiritWolf1.CancelGCDTimer(sim)
+	SpiritWolves.SpiritWolf2.CancelGCDTimer(sim)
+}
+
+// Source: https://web.archive.org/web/20201120214816/https://github.com/dalaranwow/dalaran-wow/issues/4670
+var spiritWolfBaseStats = stats.Stats{
+	stats.Agility:     113,
+	stats.Strength:    331,
+	stats.AttackPower: 836, // uncertain number. 1 STR = 2AP, but surely AGI has a contribution?. This is copy-value from dk/ghoul
+
+	// Add 1.8% because pets aren't affected by that component of crit suppression.
+	stats.MeleeCrit: (1.1515 + 1.8) * core.CritRatingPerCritChance,
+}
+
+func (shaman *Shaman) NewSpiritWolf(index int) *SpiritWolf {
+	spiritWolf := &SpiritWolf{
+		Pet: core.NewPet(
+			"Spirit Wolf "+strconv.Itoa(index),
+			&shaman.Character,
+			spiritWolfBaseStats,
+			shaman.makeStatInheritance(),
+			false,
+		),
+		shamanOwner: shaman,
+	}
+
+	spiritWolf.EnableAutoAttacks(spiritWolf, core.AutoAttackOptions{
+		MainHand: core.Weapon{
+			BaseDamageMin:  218,
+			BaseDamageMax:  311,
+			SwingSpeed:     1.5,
+			SwingDuration:  time.Millisecond * 1500,
+			CritMultiplier: 2,
+		},
+		AutoSwingMelee: true,
+	})
+
+	spiritWolf.AddStatDependency(stats.Strength, stats.AttackPower, 1.0+2)
+	spiritWolf.AddStatDependency(stats.Agility, stats.MeleeCrit, 1.0+(core.CritRatingPerCritChance/83.3))
```

Comment:
I read it was 1% crit  = 83.3 agi rating somewhere on the internett. This is just a guessimate.

Reply:
83.3 agi = 1% crit is correct

Reply:
Incorrect. Wolves scale with 2 ap = 1 str.

They only scale 1-1 from AP inherited from the shaman owner. This line is not what implements that.

In other words, Kibler's bits will yield an additional 40 attack power for the wolves not accounting for the rest of the raid buffs.